src/model/engines/engine_base.nit

```diff
@@ -36,30 +36,21 @@ class Engine
 
 		# Not yet completed mission.
 		if submission.mission.testsuite.is_empty then
-			submission.status = "error"
 			submission.compilation.title = "Work in progress"
 			submission.compilation.message = "There is no tests for the mission yet.\nPlease try again later."
```

Comment:
Oups, didn't see that one one the first review:

> There is no test~~s~~

And while we are at it, may I suggest:

> There is no test for **this** mission yet.